web/src/index.css

```
@@ -234,10 +244,15 @@
     font-family: var(--font-sans);
-    /* Handle safe areas for iOS devices (except bottom - handled by MobileFooterNav) */
-    padding-top: env(safe-area-inset-top);
-    padding-left: env(safe-area-inset-left);
-    padding-right: env(safe-area-inset-right);
+    /* Handle safe areas for iOS devices (except bottom - handled by MobileFooterNav).
+       !important: react-remove-scroll zeroes body padding while a Radix modal is open. */
+    padding-top: env(safe-area-inset-top) !important;
+    padding-left: env(safe-area-inset-left) !important;
+    padding-right: env(safe-area-inset-right) !important;
+    /* landscape: the user picks the camera side, don't reserve the right inset */
+    @variant phone-land {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)